source/contracts/Controller.sol

```diff
@@ -24,48 +24,43 @@ contract Controller is IController {
     address public owner;
     mapping(address => bool) public whitelist;
     mapping(bytes32 => ContractDetails) public registry;
+    bool public stopped = false;
```

Comment:
I prefer this way, but I thought the coding style required private variables with public getters?

Reply:
This is just a return to how it was previously implemented. It didn't ever follow our coding style but I don't feel strongly about making it conform.